inc/customizer/customizer-settings.php

```diff
@@ -37,475 +45,520 @@ public function render_content() {                  ?>
 			}
 		}
 
-		// ==============================================================
-		// ==============================================================
-		// = Remove Default WordPress Customizer Controls and Sections  =
-		// ==============================================================
-		// ==============================================================
+		// Remove default sections and controls we do not need/want.
 		$wp_customize->remove_section( 'background_image' );
 		$wp_customize->remove_section( 'colors' );
 		$wp_customize->remove_section( 'header_image' );
 
+		$wp_customize->remove_control( 'blogdescription' );
 		$wp_customize->remove_control( 'site_icon' );
```

Comment:
Do we also need to remove blogdescription field from **Settings >> General** section?

Reply:
Good question. It was removed because there was no use for it in the new Web Standards (it used to be displayed in the old ASU footer.) 

But if you guys have a good use case for it, go ahead and bring it back it. If so, we need to document where the Blog Description property is to be displayed in the site framework. It is not actually rendered anywhere right now.

Reply:
I think Zainab is asking if we should add code to _also_ remove this field from Dashboard->Settings->General. While we have removed this setting from the Customizer, it can still be set via general settings. I don't know if it's expected behavior for a theme to remove standard admin controls like that, but since we're not actually using this value, I can see how it might confuse someone if they set the tagline and it has no effect.

Reply:
Ah, brilliant!